resources/todomvc/architecture-examples/backbone/scripts/build.js

```diff
@@ -0,0 +1,56 @@
+const fs = require("fs").promises;
+
+const rootDirectory = "./";
+const sourceDirectory = "./src";
+const targetDirectory = "./dist";
+
+const htmlFile = "index.html";
+
+const filesToMove = ["node_modules/todomvc-common/base.css", "node_modules/todomvc-app-css/index.css", "node_modules/jquery/dist/jquery.min.js", "node_modules/underscore/underscore-min.js", "node_modules/backbone/backbone-min.js"];
+
+const copy = async (src, dest) => {
+    await fs.copyFile(src, dest);
+};
+
+const build = async () => {
+    // remove dist directory if it exists
+    await fs.rm(targetDirectory, { recursive: true, force: true });
+
+    // re-create the directory.
+    await fs.mkdir(targetDirectory);
+
+    // copy src folder
+    await fs.cp(sourceDirectory, targetDirectory, { recursive: true }, (err) => {
+        if (err) console.error(err);
+    });
+
+    // copy html file
+    await fs.copyFile(`${rootDirectory}${htmlFile}`, `${targetDirectory}/${htmlFile}`);
+
+    // copy files to move
+    for (let i = 0; i < filesToMove.length; i++) {
+        const fileName = filesToMove[i].split("/").pop();
+        await copy(filesToMove[i], `${targetDirectory}/${fileName}`);
+    }
+
+    // read html file
+    let html = await fs.readFile(`${targetDirectory}/${htmlFile}`, "utf8");
+
+    // remove base paths from files to move
+    for (let i = 0; i < filesToMove.length; i++) {
+        const basePath = `${filesToMove[i].split("/").slice(0, -1).join("/")}/`;
+        html = html.replace(basePath, "");
+    }
+
+    // remove basePath from source directory
+    const basePath = `${sourceDirectory.split("/")[1]}/`;
```

Comment:
Globally replacing `src/` in the file feels like it could cause a bug in the general case (I realize we are working in a pretty constrained environment here - but if we want a script like this for other apps it's probably worth attention). Perhaps replacing `="src/` and/or `="./src/` would prevent false positives? Don't feel too strongly here - at some point it may make sense to just use webpack/parcel/etc as well, but I'm OK with a simple script for the Backbone case since that's may be how these apps are deployed.

Reply:
The script has limitations, but so far worked as expected for me. 
If we need something more flexible, we can always replace it with something you mentioned above.